src/main/java/io/supertokens/bulkimport/BulkImport.java

```diff
@@ -84,7 +143,377 @@ public static BulkImportUserPaginationContainer getUsers(AppIdentifier appIdenti
         return new BulkImportUserPaginationContainer(resultUsers, nextPaginationToken);
     }
 
-    public static List<String> deleteUsers(AppIdentifier appIdentifier, Storage storage, String[] userIds) throws StorageQueryException {
+    public static List<String> deleteUsers(AppIdentifier appIdentifier, Storage storage, String[] userIds)
+            throws StorageQueryException {
         return StorageUtils.getBulkImportStorage(storage).deleteBulkImportUsers(appIdentifier, userIds);
     }
+
+    public static long getBulkImportUsersCount(AppIdentifier appIdentifier, Storage storage,
+            @Nullable BULK_IMPORT_USER_STATUS status)
+            throws StorageQueryException {
+        return StorageUtils.getBulkImportStorage(storage).getBulkImportUsersCount(appIdentifier, status);
+    }
+
+    public static synchronized AuthRecipeUserInfo importUser(Main main, AppIdentifier appIdentifier,
```

Comment:
there is a lot of code repetition between this func and the processUser func in the cronjob. Please refactor.

Reply:
Refactored it.